magda-storage-api/src/test/storage.spec.ts

```diff
@@ -77,6 +77,49 @@ describe("Storage API tests", () => {
         }
     });
 
+    describe("Create bucket", () => {
+        // Random string
+        const dummyBucket =
+            Math.random()
+                .toString(36)
+                .substring(2, 15) +
+            Math.random()
+                .toString(36)
+                .substring(2, 15);
+        it("Not an admin", () => {
+            return mockAuthorization(
+                authApiUrl,
+                false,
+                jwtSecret,
+                request(app)
+                    .post("/v0/create/bucket")
+                    .query({
+                        bucket: dummyBucket
+                    })
+                    .expect(401, "Not authorized.")
+            );
+        });
+
+        it("As an admin", () => {
+            return mockAuthorization(
+                authApiUrl,
+                true,
+                jwtSecret,
+                request(app)
+                    .post("/v0/create/bucket")
+                    .query({
+                        bucket: dummyBucket
+                    })
+                    .expect(200, {
+                        message:
+                            "Bucket " +
+                            dummyBucket +
+                            " created successfully in unspecified-region 🎉"
+                    })
+            );
+        });
```

Comment:
Should probably have a test for creating a bucket that already exists

Reply:
Added a test for that.